frontend/src/pages/OrgProfile.js

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState}from "react";
+
+const OrgProfile = () => {
+  const [org, setOrg] = useState("");
+
+  useEffect(() => {
+    const fetchOrg = async () => {
+      const orgId = window.location.pathname.split('/')[2];
+      const response = await fetch('/api/orgUsers/' + orgId);
```

Comment:
Line 9 are you able to access the backend the `fectch('api/orgUsers/' + orgId)`? I think I had to do string interpolation to work `/api/orgUsers/${body._id}` for it to work. But if the above works for you ignore me :)